include/nuttx/wireless/ieee80211/bcmf_gspi.h

```diff
@@ -0,0 +1,130 @@
+/****************************************************************************
+ * include/nuttx/ieee80211/bmcf_gspi.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __INCLUDE_NUTTX_WIRELESS_IEEE80211_BCMF_GSPI_H
+#define __INCLUDE_NUTTX_WIRELESS_IEEE80211_BCMF_GSPI_H
```

Comment:
could we split the bcmf related change to new PR?

Reply:
Sure.  

To be clear I will pull all the actual Pico-W related changes in the "boards" and "arch/arm" directories and resubmit the PR as "adding gSPI support to bcmf".  I will submit a separate PR for "adding pico-w".

Does this match your intent?

Reply:
Yes, the split looks reasonable.